src/plugins/credential_management/README.md

```diff
@@ -10,16 +10,34 @@ See the [OpenSearch Dashboards contributing
 guide](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/master/CONTRIBUTING.md) for instructions
 setting up your development environment.
 
+## Configuration
 
-## Build and Run crypto_materials_generator
+1. To enable this feature, override config/opensearch_dashboards.yml
 
 ```
-npm install @types/yargs
-npm install -g ts-node typescript '@types/node'
+credential_management.enabled: true
```

Comment:
we need a server level config to enable/disable the multi-datasource feature, not a credentials management plugin config

Reply:
Yes, I would like to split the global config change to a seperate PR as it would be used for conditional rendering. Tracking https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1904

Reply:
replaced with global config